src/azul/__init__.py

```diff
@@ -183,6 +183,13 @@ def data_portal_name(self):
     def dss_endpoint(self) -> Optional[str]:
         return os.environ.get('AZUL_DSS_ENDPOINT')
 
+    def canned_sources(self, catalog: CatalogName) -> AbstractSet[str]:
+        try:
+            sources = os.environ[f'AZUL_CANNED_{catalog.upper()}_SOURCES']
```

Comment:
No modifications to the `env` files that go with this?
dev/sandbox or descriptions in the project_root `env.py` file?